lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -178,9 +172,6 @@ class MongoClientInterface {
             _id: bucketName,
         }, {}, (err, doc) => {
             if (err) {
-                log.error(
```

Comment:
Is there a reason why the error logs are being removed?

Reply:
this branch was pulled off an older version of Arsenal, I think, rebasing off master probably didn't rebase correctly, fixing now